doc/man/8/radosgw-admin.rst

```diff
@@ -566,9 +566,13 @@ Options
 
 .. option:: --shard-id=<shard-id>
 
-	Optional for mdlog list. Required for ``mdlog trim``,
+	Optional for mdlog list, data sync status. Required for ``mdlog trim``,
 	``replica mdlog get/delete``, ``replica datalog get/delete``.
 
+.. option:: --max-entries=<entries>
+
+	Optional for data sync status, default value is 20.
+
```

Comment:
`--max-entries` is used in several places, most of which use a default of 1000. i'm not sure it's worth spelling it out for each command though.. maybe just use something generic like `max entries for listing operations`